apps/scan/frontend/src/screens/poll_worker_screen.tsx

```diff
@@ -169,268 +332,191 @@ export function PollWorkerScreen({
 
   const usbDriveStatus = usbDriveStatusQuery.data;
   const printerStatus = printerStatusQuery.data;
-  const needsToAttachPrinterToTransitionPolls = !printerStatus?.connected;
+  const isPrinterReady = Boolean(printerStatus?.connected);
+  const pollsInfo = pollsInfoQuery.data;
+  const { pollsState } = pollsInfo;
+  const lastPollsTransition =
+    pollsInfo.pollsState === 'polls_closed_initial'
+      ? undefined
+      : pollsInfo.lastPollsTransition;
 
   function showAllPollWorkerActions() {
     return setPollWorkerFlowState(undefined);
   }
 
-  async function transitionPolls(pollsTransitionType: PollsTransitionType) {
-    try {
-      // In compliance with VVSG 2.0 1.1.3-B, confirm there are no scanned
-      // ballots before opening polls, even though this should be an impossible
-      // state in production.
-      if (pollsTransitionType === 'open_polls' && scannedBallotCount > 0) {
-        setIsShowingBallotsAlreadyScannedScreen(true);
-        await logger.log(LogEventId.PollsOpened, 'poll_worker', {
-          disposition: 'failure',
-          message:
-            'Non-zero ballots scanned count detected upon attempt to open polls.',
-          scannedBallotCount,
-        });
-        return;
-      }
-
-      setPollWorkerFlowState('polls_transition_processing');
-
-      const pollsTransitionTime = getCurrentTime();
-      await transitionPollsMutation.mutateAsync({
-        type: pollsTransitionType,
-        time: pollsTransitionTime,
+  async function openPolls() {
+    // In compliance with VVSG 2.0 1.1.3-B, confirm there are no scanned
+    // ballots before opening polls, even though this should be an impossible
+    // state in production.
+    if (scannedBallotCount > 0) {
+      setIsShowingBallotsAlreadyScannedScreen(true);
```

Comment:
This could also move to the backend at some point and just show an error message if it occurs

Reply:
True. We have the assertion on the backend but we could just pass an `err` `Result` and render the screen accordingly. I'd like that because it would move this logging to the backend, but punted on it here.